dotnet-471-dev-pack/README.md

```diff
@@ -0,0 +1,172 @@
+# Habitat package: dotnet471
+
+## Description
+
+This package is for the .net 4.7.1 framework developer pack from microsoft. It provides 4.7.1 reference assemblies as well as the sdk needed to support building .net 4.7.1 applications
+
+## Maintainers
+
+The Habitat Maintainers humans@habitat.sh
+
+## Type of Package
+
+Binary
+
+## Usage
+
+Add core/dotnet471 to pkg_build_deps then leverage nuget and msbuild to build and deploy your application. For an example see mwrock's asp full framework MVC blog post: https://www.habitat.sh/blog/2017/08/Habitat-plans-for-net-full-iis-apps/
+
+## Notes
+
+# Testing dotnet-471-dev-pack on Windows
+
+**Note**: The following manual steps can be performed in one automated step: [see these instructions](#run-the-tests-via-powershell-script)
+
+## Build and Install
+
+### Build the dotnet-471-dev-pack core plan 
+
+Ensure that:
+* the [core-plan repo](https://github.com/habitat-sh/core-plans) is cloned locally
+* a terminal is opened to the root of the repo
```

Comment:
There's no need to cover doing a git clone. Lines 29-37 here can simply be reduced to stating that steps given are from the root directory.

Reply:
Fixed